fe/fe-core/src/main/java/org/apache/doris/task/StreamLoadTask.java

```diff
@@ -320,6 +328,9 @@ private void setOptionalFromTSLPutRequest(TStreamLoadPutRequest request) throws
         if (request.isSetLoadToSingleTablet()) {
             loadToSingleTablet = request.isLoadToSingleTablet();
         }
+        if (request.isSetHiddenColumns()) {
+            hiddenColumns = Arrays.asList(request.getHiddenColumns().split(","));
```

Comment:
Need to trim the space, eg:
`"delete, sequence" => "delete,sequence"`

Reply:
fixed